backend/balboa-rocksdb/main.c

```diff
@@ -29,11 +29,9 @@ Usage: balboa-rocksdb [options]\n\
     -S disable signal handling\n\
     -R disable engine stats reporter\n\
     -j connection throttle limit, maximum concurrent connections (default: 64)\n\
-    --membudget <memory-in-bytes> rocksdb membudget option (value: %" PRIu64
-      ")\n\
+    --membudget <memory-in-bytes> rocksdb membudget option (value: %zu)\n\
     --parallelism <number-of-threads> rocksdb parallelism option (value: %d)\n\
-    --max_log_file_size <size> rocksdb log file size option (value: %" PRIu64
```

Comment:
actually i think `max_log_file_size` should be `uint64_t` to allow log files bigger than 2GiB (or 4GiB). please check the definition of `max_log_file_size` in rocksdb.

Reply:
It's `size_t` there, so let's keep it the way it is.

Reply:
agreed